.travis.yml

```diff
@@ -1,13 +1,16 @@
 language: python
 python:
-  - "3.6"
+  - "3.6.2"
 
 os:
   - linux
 
+env:
+  - PIPENV_IGNORE_VIRTUALENVS=1
```

Comment:
This is needed in order to drop a message [1] on Travis. 
[1] `Courtesy Notice: Pipenv found itself running within a virtual environment, 
so it will automatically use that environment, instead of 
creating its own for any project.`